RealmModelGenerator/AttributesRelationshipsViewController.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  AttributesRelationshipsViewController.swift
+//  RealmModelGenerator
+//
+//  Created by Zhaolong Zhong on 3/31/16.
+//  Copyright © 2016 QuarkWorks. All rights reserved.
+//
+
+import Cocoa
+
+@objc protocol AttributesRelationViewControllerDelegate {
+    optional func attributesRelationshipsViewController(attributesRelationshipsViewController:AttributesRelationshipsViewController, selectionChange index:Int)
+}
+
+class AttributesRelationshipsViewController: NSViewController, AttributesViewControllerDelegate {
+    static let TAG = NSStringFromClass(AttributesRelationshipsViewController)
+    
```

Comment:
Since our names for our view controllers are so long we should Replace ViewController with VC. I did this in my branch if you would like to see an example